commands/members.js

```diff
@@ -0,0 +1,7 @@
+module.exports.run = async function () {
+    return this.reply(`**Member Count:** ${message.guild.members.cache.filter((m) => !m.bot).size.toLocaleString()}`);
+}
+
+module.exports.info = {
+    aliases: ['members']
```

Comment:
```suggestion
```
Aliases don't need to be added when the file name is already the command
(Do the same for avatar command)